Blockchain/Contribute/README.md

```diff
@@ -40,3 +40,5 @@
 - [Web3j](https://github.com/web3j/web3j)
 - [Hardhat](https://github.com/NomicFoundation/hardhat)
 - [Foundry](https://github.com/foundry-rs/foundry)
+- [Remix Project](https://github.com/ethereum/remix-project)
+
```

Comment:
please remove this extra line

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix